### PR TITLE
Test reader.ReadLag() to return zero lag when using consumer group.

### PR DESCRIPTION
**Pull request by ardhitama**

### CODE REVIEW

File: reader_test.go [hunk 1]
```diff
@@ -570,6 +570,19 @@ func TestReaderLagWhenConsumerGroupsEnabled(t *testing.T) {
 	}
 }
 
+func TestReaderReadLagReturnsZeroLagWhenConsumerGroupsEnabled(t *testing.T) {
+	r := &Reader{config: ReaderConfig{GroupID: "not-zero"}}
+	lag, err := r.ReadLag(context.Background())
+
+	if err != errNotAvailableWithGroup {
+		t.Fatalf("expected %s; got %s", errNotAvailableWithGroup.Error(), err.Error())
```
Comment (nlsun) [inline reader_test.go:578]: no need to call `.Error()` on these

Comment (nlsun) [inline reader_test.go:578]: no need to call .Error() on these

**Review by nlsun — COMMENTED** [review]
small comment, otherwise lgtm

File: reader_test.go (hunk 1 above)
Comment (ardhitama) [reply reader_test.go]: changed to use %v

**Review by nlsun — APPROVED** [review]

**Pull request merged**